examples/swaps-api-nextjs-bitcoin/src/components/Swap.tsx

```diff
@@ -252,46 +308,58 @@ const Swap = () => {
        * In this excerpt, here's how to
        */
 
-      // if (transfer.tx.meta) { <- For Bitcoin to ETH, the send endpoint will return an object called `meta`
-      //   const { from, recipient, amount, memo } = transfer.tx.meta;
-
-      //   (window.xfi as any)?.bitcoin.request( <- Here, we're prompting a users wallet using xDEFI injected SDK
-      //     {
-      //       method: "transfer",
-      //       params: [
-      //         {
-      //           from,
-      //           recipient,
-      //           amount,
-      //           memo,
-      //         },
-      //       ],
-      //     },
-      //     (error: any, result: any) => {
-      //       console.debug(error, result);
-      //     },
-      //   );
-
-      //   txData = {
-      //     from,
-      //     to: recipient,
-      //     amount: amount,
-      //     data: memo,
-      //   };
-      // } else {
-
-      // }
-
       setTransStatus({ status: "Wallet Interaction Required" });
 
-      const txResponse = await signer?.sendTransaction(txData);
-
-      pollTransactionStatus(transfer.id.toString(), txResponse?.hash!);
+      let txResponse;
+
+      if (transfer?.tx.meta) { 
+        // For Bitcoin to ETH, the send endpoint will return an object called `meta`
+        
+        const { from, recipient, amount, memo } = transfer.tx.meta;
+        
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (window.xfi as any)?.bitcoin.request( 
+          // Here, we're prompting a users wallet using xDEFI injected SDK
+          {
+            method: "transfer",
+            params: [
+              {
+                from,
+                recipient,
+                amount,
+                memo,
+              },
+            ],
+          },
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (error: any, result: any) => {
+            console.log(error, result);
+
+            if(error) {
+              toast({
+                variant: "destructive",
+                title: "Something went wrong!",
```

Comment:
Looks good to me. It will be awesome if we can throw a detail message.

Reply:
Alright, got it!

Reply:
Here is quite tricky as the error message returned is a number if I remember correctly.